src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -31,6 +31,26 @@ function dump($var, ...$moreVars)
     }
 }
 
+if (!function_exists('dumpif')) {
+    /**
+     * @author Tales Santos <tales.augusto.santos@gmail.com>
+     */
+    function dumpif($var, ...$moreVars)
```

Comment:
I would instead use the signature `dumpIf(callable $condition, $var, ...$moreVars)`

Reply:
I prefer the original signature in this case. Is better to see the $var being dump first and then the callable.